.github/workflows/reusable-jira-workflow.yml

```
@@ -22,9 +22,9 @@ jobs:
             }
 
             Set-JiraConfigServer -Server "${{ secrets.JIRA_URL }}"
-            if("${{ inputs.branch_name }}" -match "(?<=\/)[a-zA-Z]+[_-][0-9]+" -eq $False)
+            if("${{ inputs.branch_name }}" -match "(?<=\/)[a-zA-Z0-9]+[_-][0-9]+" -eq $False)
```

Let's make sure a project name starts with a letter

```suggestion
            if("${{ inputs.branch_name }}" -match "(?<=\/)[a-zA-Z][a-zA-Z0-9]+[_-][0-9]+" -eq $False)
```

---

Makes sense to me!